diff --git a/node_modules/@discordjs/voice/dist/index.js b/node_modules/@discordjs/voice/dist/index.js
index 7ddadee..61609fa 100644
--- a/node_modules/@discordjs/voice/dist/index.js
+++ b/node_modules/@discordjs/voice/dist/index.js
@@ -1358,6 +1358,7 @@ var VoiceConnection = class extends import_node_events7.EventEmitter {
     }
   }
   addStatePacket(packet) {
+    this.configureNetworking();
     this.packets.state = packet;
     if (typeof packet.self_deaf !== "undefined")
       this.joinConfig.selfDeaf = packet.self_deaf;
